.github/ISSUE_TEMPLATE/type_proposal.md

```diff
@@ -0,0 +1,26 @@
+---
+name: "Proposal"
+about: "Start a proposal for a new feature or an improvement"
+labels: 'Type/Proposal'
+---
+
+## Summary
+<!-- REQUIRED -- Provide a short summary of the proposal, at most one or two sentences -->
+
+## Goals
+<!-- What are the goals of this proposal?  Omit this section if you have
+nothing to say beyond what's already in the summary. -->
+
+## Non-Goals
+<!-- Describe any goals you wish to identify specifically as being out of
+scope for this proposal. -->
+
+## Motivation
+
+<!-- Why should this work be done?  What are its benefits?  Who's asking
+for it?  How does it compare to the competition, if any? -->
+
+## Description
+
+<!-- REQUIRED -- Describe the enhancement in detail: Both what it is and,
```

Comment:
Great! since we follow the JEP template shall we add the rest of them as well. 

https://openjdk.java.net/jeps/2

Reply:
Fixed in 22fc6d4